shopfloor/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 {
     "name": "Shopfloor",
     "summary": "manage warehouse operations with barcode scanners",
-    "version": "16.0.2.8.0",
+    "version": "18.0.1.0.0",
```

Comment:
Wondering if we shouldn't have a 18.0.0.x.y as we didn't _fully_ migrate yet :question:

Reply:
done 0.1.0